Intersect.Editor/Configuration/ToolCursor.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using Intersect.Editor.General;
+
+namespace Intersect.Editor.Configuration
+{
+    public sealed class ToolCursor
+    {
+        public Point CursorClickPoint { get; set; }
+
+        private static readonly Dictionary<EditingTool, ToolCursor> _toolCursorDict;
+
+        public static IReadOnlyDictionary<EditingTool, ToolCursor> ToolCursorDict => _toolCursorDict;
+
+        static ToolCursor()
+        {
+            _toolCursorDict = new Dictionary<EditingTool, ToolCursor>
+            {
+                { EditingTool.Brush, new ToolCursor() },
+                { EditingTool.Dropper, new ToolCursor() },
+                { EditingTool.Erase, new ToolCursor() },
+                { EditingTool.Fill, new ToolCursor() },
+                { EditingTool.Rectangle, new ToolCursor() },
+                { EditingTool.Selection, new ToolCursor() }
+            };
+        }
+
+        public static void Load()
+        {
+            ToolCursor toolCursor = new ToolCursor();
```

Comment:
Don't initialize here, just declare it. It needs to get recreated each time the `else` branch happens otherwise there will be multiple copies of the same `ToolCursor` in the dictionary.